### PR TITLE
add a SECURITY.md

### PR DESCRIPTION
**Pull request by jschauma**

### CODE REVIEW

**Review by wbl — APPROVED** [review]
Squash down and I will merge!

**Comment by jschauma** [comment]
Ok, I think I did?

**Comment by richsalz** [comment]
I find the easiest thing is "git rebase --squash". Or "rebase -i" and squash by editing the commit list.

**Comment by jschauma** [comment]
I thought that's what I did there.  Pretty much operating in xkcd 1597 mode here...

**Review by wbl — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -35,6 +35,15 @@ issue appeals to you, feel free to jump in and get started on it.
 However, there's a bunch of design stuff that also needs to happen, but we'll
 make sure to have the conversation in an open manner.
 
+Where/how can I report vulnerabilities?
+---------------------------------------
+
+Please see our [this
+document](https://github.com/quictls/quictls/blob/main/SECURITY.md)
```
Comment (botovq) [inline README.md:42]: Should 'this document' be 'security policy'? Or spell it out fully: 'security policies or procedures'? Or (worst option in my opinion) drop 'our'.

Comment (jschauma) [reply README.md:42]: Good call. :-)

https://github.com/quictls/quictls/pull/54

**Pull request merged**